.github/workflows/ci.yaml

```diff
@@ -22,51 +48,47 @@ jobs:
           node-version: 18
 
       - name: Install Clean dependencies
-        working-directory: ./angular
         run: npm ci
 
       - name: Run headless tests
         id: run-tests
-        run: npm test -- --watch=false --browsers=ChromeHeadless 
-        working-directory: ./angular
-        continue-on-error: true
+        if: ${{ always() }}
+        run: npm run test -- --watch=false --browsers=ChromeHeadless 
```

Comment:
Why did you add these args to tests?

Reply:
`--watch=false` disables the watch mode, which is unnecessary in CI/CD environments where tests should run only once.
`--browsers=ChromeHeadless` runs tests in a headless Chrome browser, which is ideal, because it doesn't require a graphical interface.